receiver/hostmetricsreceiver/internal/scraper/raidscraper/metadata.yaml

```diff
@@ -0,0 +1,70 @@
+type: raid
+
+status:
+  class: scraper
+  stability:
+    development: [metrics]
+  distributions: [core, contrib, k8s]
+  codeowners:
+    active: [dmitryax, braydonk]
+
+sem_conv_version: 1.9.0
+
+attributes:
+  device:
+    description: Name of the disk.
+    type: string
+  state:
+    description: State of the disk
+    type: string
+
+metrics:
+  md.blocks.total:
```

Comment:
What does `md` stands for? It sounds not specific enough and potentially can cause conflicts going forward. Please take a look at the [naming guidelines](https://opentelemetry.io/docs/specs/semconv/general/naming/).

Also, we are in process of stabilizing [system metrics](https://github.com/open-telemetry/semantic-conventions/tree/main/docs/system) in OTel Semantic Conventions. Ideally we need to define them in the Semantic conventions first before adding them to the receiver. Otherwise, they will be more likely to experience breaking changes in the future. So it's better to start there with an issue and PR to get feedback from the SemConv group sooner.

Reply:
Its this https://manpages.ubuntu.com/manpages/trusty/man4/md.4.html Dimitri.
So multipledevice I suppose. 
I will raise the issue/PR for naming as suggested but multipledevice seems a bit long and I can't think of anything other than md to call it.

Reply:
Also I am now thinking maybe this sits closer to the naming convention here https://github.com/open-telemetry/semantic-conventions/blob/main/docs/system/hardware-metrics.md#hwdisk_controller---disk-controller-metrics.
Is there a receiver for disk controller already? I cant find one in the contrib repo